### DISCUSSION TITLE
Some comments on the new format?

### DISCUSSION THREAD

**Discussion by davidhcefx** [davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#13] [Ideas]
Hello,

- First of all, I plan to change each chapters to [tabular formats][tabular], because I can't think of a better representation if we were to add kart info 😆 (see Discussions)
- Then, I think that maybe merging [chapters 37-40, 41-44, ..., 61-64][merged] would be better for navigation. No?
- For the Bonus chapters, I am thinking of three options regarding the numberings:

A. [Original][]: Multiple "Bonus" words which seems redundant...
```md
### Bonus
- Bonus 1: Track Name
- Bonus 2: Track Name
```

B. Zero prefixed
```md
### Bonus
- 0-1: Track Name
- 0-2: Track Name
```

C. ["B" prefixed][]
```md
### Bonus
- B-1: Track Name
- B-2: Track Name
```

Some comments? @FauxBlue 

[tabular]: https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode/blob/40f891ac25adfcf81cdfc2ff5a5486e5ce039dd5/README.md#future-factory
[merged]: https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode/blob/40f891ac25adfcf81cdfc2ff5a5486e5ce039dd5/README.md#chapter-37-40
[Original]: https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode/blob/40f891ac25adfcf81cdfc2ff5a5486e5ce039dd5/README.md#bonus-1
["B" prefixed]: https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode/blob/40f891ac25adfcf81cdfc2ff5a5486e5ce039dd5/README.md#bonus

**Comment by FauxBlue** [comment]
Yes, definitely, to the tabular format. Looks a lot better.
As a side note, would it be worth separating the Theme into another column? (The brackets after each track name.)

---

I don't really see how merging the chapters would be better for navigation. Is it about grouping them the same as the in game categories page? e.g. Chapters 37-40 are the Season 4 China story.
![Screenshot_2022-05-16-19-35-11-642_com nexon kart](https://user-images.githubusercontent.com/33422761/168661092-9c2f2759-6488-404d-a787-c71b3b041449.jpg)
 
I think being able to link to individual chapters is best.

---

Yes, I wasn't entirely comfortable with the repetition when I wrote it, but "Bonus" seemed like the most suitable heading, and I then just copied the in-game numbering.

I prefer the look of option C ("B" prefixed).

**Reply by davidhcefx** [reply]
Thanks for your input!

I’m not sure whether should we separate the Theme to its own column. Because I noticed that the theme always come right after the name, so I suspect it’s part of the track name…?

The reason I think merging chapters (including those before 37-40) might be beneficial is that there are a bit too many items within the table of contents (the hamburger button). But it’s fine going either way :)

**Reply by FauxBlue** [reply]
I'm not too fussed about separating the theme. The track names are always presented in-game with their theme in the same format, so in my mind they are logically two different fields, but it won't really make a difference for the purposes of this list so no need to change it. I think the biggest influence in me suggesting it comes from [this website](https://krrplus.web.app/tracks), where separating them at least serves a purpose in that you can sort by the theme (i.e. view them grouped together).

I see what you mean (, I had never looked at the table of contents). I would have argued there's a case for being able to directly link to a specific chapter (e.g. [#chapter-9](https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#chapter-9), [#chapter-34](https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#chapter-34), [#chapter-55](https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#chapter-55)) which is more intuitive, although my argument kind of falls apart now that there are more stories so Future Factory Chapter 1 becomes [#chapter-1-1](https://github.com/davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#chapter-1-1). I still think keeping them separate is better visually as I find it easier to locate the one level I am looking for with the headings working as a separator. But, even if I don't use the table of contents, yes, having cleaner navigation would be good.

Not sure if there is a best-of-both-worlds third option. Does GitHub let you control the max depth of the table of contents? I couldn't find anything to suggest they do with a quick google, but if they did then nesting them one further might have worked. e.g:

> \### Chapters 37-40 (S4 China)
> 
> \#### Chapter 37
> 
> [table]
> 
> \#### Chapter 38
> 
> etc.

With the table of contents limited to 3 deep, the individual chapters would then not show on the table of contents. But without the ability to limit the depth then this would only make your issue worse.

**Reply by davidhcefx** [reply]
Thanks for mentioning that unofficial manual! It's so hard to land there with Google search... 😅
